IdentityCore/src/util/NSString+MSIDExtensions.m

```diff
@@ -259,14 +259,21 @@ - (BOOL)msidIsEquivalentWithAnyAlias:(NSArray<NSString *> *)aliases
     {
         return NO;
     }
-
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Check if credential environment %@ is included in the list of valid environment aliases.",self);
+    for (NSString *alias in aliases)
+    {
+        MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, nil, @"environment alias found: %@",alias);
```

Comment:
Can we do this logging below on line 269 to avoid extra loop?

Reply:
That's a good point. However, the next for loop returns as soon as the conditional gets satisfied, so we may not get to print out all of the available environment aliases.